src/ucm/mmap/install.c

```diff
@@ -16,18 +16,30 @@
 #include <ucm/util/log.h>
 #include <ucm/util/reloc.h>
 #include <ucm/util/sys.h>
+#include <ucm/bistro/bistro.h>
 #include <ucs/sys/math.h>
 
 #include <sys/mman.h>
 #include <sys/shm.h>
 #include <unistd.h>
 #include <pthread.h>
 
+#define UCM_IS_HOOK_ENABLED(_entry)                                                      \
+    (((_entry)->hook_type == UCM_HOOK_BOTH) ||                                           \
+     (!ucm_global_opts.enable_bistro_hook && ((_entry)->hook_type == UCM_HOOK_RELOC)) || \
+     (ucm_global_opts.enable_bistro_hook && ((_entry)->hook_type == UCM_HOOK_BISTRO)))
+
+typedef enum ucm_mmap_hook_type {
+    UCM_HOOK_BOTH   = 0,
```

Comment:
maybe it should be flags ? and don't rely on default initialization to 0 ;)

Reply:
fixed